overlays/imx8qm_display_vm.nix

```diff
@@ -0,0 +1,9 @@
+final: prev: {
+  
+  ubootImx8 = prev.ubootImx8.overrideAttrs (old: {
+    patches = old.patches ++ [
+      ./bsp/u-boot/imx8qm/patches/0001-Add-support-for-Display-VM.patch
```

Comment:
Let's put this patch to the imx-uboot.nix file directly instead?

Reply:
Due this patch should be applied only for the imx8qm-display-vm-config.nix configuration, it is necessary that this patch stays in an overlay.

Reply:
Yes, now I see, thanks!